x/oracle/test_common.go

```diff
@@ -38,8 +38,8 @@ var (
 		sdk.AccAddress(pubKeys[2].Address()),
 	}
 
-	uSDRAmt  = sdk.NewInt(1005 * assets.MicroUnit)
```

Comment:
doesn't make much difference in the grand scheme of things, but why the revert to `m`-prefixed units here instead of `u`?

Reply:
That looks like an artifact of a failed rebase. Thanks for pointing that out.